packages/generator-langium/langium-template/package.json

```diff
@@ -44,7 +44,7 @@
     },
     "dependencies": {
         "chevrotain": "^9.1.0",
-        "colors": "^1.4.0",
+        "chalk": "^5.0.1",
```

Comment:
Suggestion: This should probably also be `^4.1.2`, since downstream users of Langium will have the same ESM issue.

Reply:
corrected the version for the template